modules/insights-operator-downloading-archive.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-operator.adoc
+
+:_content-type: PROCEDURE
+[id="insights-operator-downloading-archive_{context}"]
+= Downloading your Insights Operator Archive
+
+Insights Operator stores gathered data in an archive located in the `openshift-insights` namespace of your cluster. You can download and review the data that is gathered by the Insights Operator. 
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Find the name of the currently running pod for the Insights Operator:
```

Comment:
You could remove "currently" to minimise this sentence.

Reply:
fixed